libertai_client/commands/agent.py

```diff
@@ -97,10 +97,22 @@ async def deploy(
             prompt=False,
         ),
     ] = None,
+    format: Annotated[
+        str | None,
+        typer.Option(
+            help="Output format for the deployment information",
+            case_sensitive=False,
+            prompt=False,
+        ),
+    ] = None,
```

Comment:
use an optional `--json` option instead that is a boolean, cf https://typer.tiangolo.com/tutorial/parameter-types/bool/#only-force